pkg/virt-controller/watch/migration.go

```diff
@@ -1107,6 +1168,12 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 		return fmt.Errorf("vmi is inelgible for migration because another migration job is running")
 	}
 
+	if err = c.handleMigrationBackoff(key, vmi, migration); err != nil {
```

Comment:
We could just move this to the specific case, right?

Reply:
Yes indeed.